source/PlayerInfo.cpp

```diff
@@ -569,10 +569,10 @@ void PlayerInfo::AddChanges(list<DataNode> &changes)
 
 
 // Add an event that will happen at the given date.
-void PlayerInfo::AddEvent(const GameEvent &event, const Date &date)
+void PlayerInfo::AddEvent(GameEvent event, const Date &date)
```

Comment:
I'm confused as to why this was changed from const
you modify the event that is received whereas before we would copy it and modify that

the event should not have it's date modified surely?

Reply:
~~Ask quyykk.~~

Reply:
The event is no longer a value-by-reference, but a value-by-copy. The copy can have the date modified. (And I'm not sure why this code is better, but I can see that it is not worse, so that is sufficient for me.)